src/latest_release_note.ts

```diff
@@ -1,21 +1,43 @@
 // TODO: find a better way of loading html into a string
-export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes 🐍 (Mar. 31, 2020)</h1>
+export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes ⌨️🐍💞 (April 15, 2020)</h1>
 <p>
-    We're back with a small hotfix!</p>
-<p>
-    <h2>Fixes:</h2>
-    <ul>
-        <li>Fixed serial port selection for serial monitor.</li>
-        <li>Fixed launch issue on Ubuntu.</li>
-    </ul>
-
-
-</p>
-<p>
-    <h2>Changes:</h2>
-    <li>Due to necessary serial monitor fixes, <b>the extension is now only supported on VS Code v1.43+</b></li>
-
-</p>
-<br>
-<p><b>Stay healthy while you hack 🥰🌼,</b><br>
-    &nbsp&nbsp&nbsp&nbsp&nbsp <b><i>- The Device Simulator Express Team</i></b></p>`;
+    We're unveiling a new addition to our DSX family of microcontroller simulators! Please welcome our <b>Adafruit CLUE
+        simulator</b> 💕🔍.<br>
+    This change is <b>hidden</b> under our preview flag by default. <a href="https://github.com/microsoft/vscode-python-devicesimulator/blob/dev/README.md#how-to-enable-preview-flag">See here</a> to learn how to enable
```

Comment:
_&ast;nit&ast;_
Consider "This change is hidden under **a** preview flag by default" instead of "This change is hidden under **our** preview flag by default".